packages/components/src/internal/components/productnavigation/ProductClickableItem.tsx

```diff
@@ -12,6 +12,7 @@ export const ProductClickableItem: FC<ProductClickableItemProps> = memo(props =>
     const onEnter = useCallback(() => setHovered(true), [setHovered]);
     const onLeave = useCallback(() => setHovered(false), [setHovered]);
 
+    // TODO: Need to use AppLinkHere, however we'd need to add several props
```

Comment:
Is this a post-PR / TBD TODO? Interesting use case since these links are rendered outside of our traditional apps.

Reply:
Yes, I think this TODO should be addressed later. Ideally we'd drop the need for onEnter/onLeave and just use CSS `:hover` to apply the different background color.

Reply:
So, there were several other issues with this component, so I did go ahead and add support for onMouseEnter and onMouseLeave to AppLink and made this component render AppLink. However, I do think we should do a follow-on PR in the near future to get rid of the hover events, and drop support for onMouseEnter/Leave (I have added comments to the code to suggest this).